vault-plugins/flant_iam/backend/factory.go

```diff
@@ -108,7 +108,7 @@ func newBackend(conf *logical.BackendConfig) (logical.Backend, error) {
 
 	logger := conf.Logger.Named("IAM")
 
-	storage.AddKafkaSource(kafka_source.NewSelfKafkaSource(mb, restoreHandlers))
+	storage.AddKafkaSource(kafka_source.NewSelfKafkaSource(mb, restoreHandlers, conf.Logger.Named("KafkaSourceSelf")))
```

Comment:
Maybe we will use `logger.Named("KafkaSourceSelf")`

Then, in logs, we will see `IAM.KafkaSourceSelf - some log`.

Iam auth has KafkaSourceSelf too